### ISSUE TITLE
Python tools do not show log messages unless --verbosity DEBUG

### ISSUE THREAD

**Issue by mbabadi** [broadinstitute/gatk#4629]
... and this leads some users to believe that GATK has frozen (I've seen several such complaints on the forum).

I use python `logger` to log INFO/WARN/DEBUG to `stdout` in `gcnvkernel`. At the moment, I do not pass GATK's log verbosity level to `gcnvkernel` (provisions for doing this is already in place -- it's just a one-liner fix to enable it).

A simple strategy is to pipe python's stdout to JVM's, regardless of GATK's log level, and delegate the responsibility setting the log level in python scripts to python tool devs.

**Comment by droazen** [comment]
@mbabadi Note that the GATK log output now goes to stderr, not stdout, to enable pipelining for select tools.

**Comment by mbabadi** [comment]
@droazen I understand that -- the issue is that `ScriptExecutor` does not print either stdout or stderr from the external process unless debug is enabled (cf. `ScriptExecutor.java` ln. 92):

```
            //if debug is enabled, output the stdout and stderr, otherwise capture it to a buffer
            if (logger.isDebugEnabled()) {
                processSettings.getStdoutSettings().printStandard(true);
                processSettings.getStderrSettings().printStandard(true);
            } else {
                processSettings.getStdoutSettings().setBufferSize(8192);
                processSettings.getStderrSettings().setBufferSize(8192);
            }
```

**Comment by samuelklee** [comment]
@droazen any reason not to fix this?

**Comment by droazen** [comment]
I defer to @cmnbroad on this one -- Chris, could you comment?

**Comment by cmnbroad** [comment]
Yes, sorry for not having gotten around to these, but yes this definitely this should be fixed.
